### DISCUSSION TITLE
How can I hide the 'Rotate Layers' button from the tool bar?

### DISCUSSION THREAD

**Discussion by ykparkwixon** [geoman-io/leaflet-geoman#930] [Questions & Help] [answered]
How can I hide the 'Rotate Layers' button from the tool bar?

Isn't there any option for that such as 'cutPolygon:false' for map.pm.addControls?

**Comment by Falke-Design** [comment, marked as answer]
There is a option: `rotateMode`:

```
map.pm.addControls({
    rotateMode: false
});
```

It's missing in the Readme, we will add it

**Comment by ykparkwixon** [comment]
> There is a option: `rotateMode`:
> 
> ```
> map.pm.addControls({
>     rotateMode: false
> });
> ```
> 
> It's missing in the Readme, we will add it

Thanks for your reply!~
